src/shared/sam/activation.ts

```diff
@@ -77,11 +77,11 @@ export async function activate(activateArguments: {
         )
     )
 
-    await detectSamCli({ showMessage: false })
+    await detectSamCli({ passive: true, showMessage: false })
     activateArguments.extensionContext.subscriptions.push(
         vscode.workspace.onDidChangeConfiguration(configurationChangeEvent => {
             if (configurationChangeEvent.affectsConfiguration('aws.samcli.location')) {
-                detectSamCli({ showMessage: undefined })
+                detectSamCli({ passive: true, showMessage: undefined })
```

Comment:
This happens when user changed their vscode settings. Is that considered active or passive?

Reply:
This seems reasonably passive